### DISCUSSION TITLE
FIL-RetroPGF-1: Nominations for Collective Governance

### DISCUSSION THREAD

**Discussion by kkarrancsu** [filecoin-project/community#694] [RetroPGF]
### Nominations for Collective Governance

This is the thread to nominate projects for the **_Collective Governance_** category of FIL-RetroPGF-1. As a reminder, this category is intended to capture work that impacted Filecoin's governance participants or helped bring new governance participants into the network.

### Nomination Template
When nominating a project, please answer the following questions in a comment to this discussion:

1. The project name
2. A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)
3. A link to the project’s GitHub or Twitter
4. Contact info for the project or project lead

### Nomination Considerations

Things to consider when nominating a project:
- Use your best effort to describe how this project has delivered impact to the Filecoin ecosystem in the previous six months (Oct 2023 - Mar 2024). Use the template provided in each category’s thread when nominating a project.
- Make sure to nominate projects in the relevant category. If you feel that a project spans multiple categories, please choose the one category that you feel it best aligns with. Do not nominate the same project across multiple categories! 
- Note that the organizers of FIL-RetroPGF-1 may choose to reassign a project’s category from its nominated one, if they feel it better aligns elsewhere. 
- Before nominating a project please check if this project has been nominated already, so we can avoid duplication.
RetroPGF is an experiment that Filecoin plans to iterate on regularly. Community feedback is key to helping the process improve with each round. We welcome your notes, reflections, and feedback at any step. Please use this discussion to submit feedback!

Project Nominations will close by Mar 31, 2024.

See [here](https://github.com/filecoin-project/community/discussions/689) for general guidelines on project nomination.

_Note: These threads will be moderated to only include project nominations. Off-topic comments that show support or hype for a project or meta-commentary on the program itself will be removed to keep the thread clean. If you want to show your excitement for a nomination, use the reaction feature!_

**Comment by NSC-FIL** [comment]
I agree with the nomination of the Asia SP Working group as described eloquently here by Fatman13:
https://github.com/filecoin-project/community/discussions/689#discussioncomment-8793892

**Comment by Fatman13** [comment]
For **Collective Governance** category, i nominate **Asia SP Working Group**. 

**1. Project Name**: Asia SP Working Group
**2. Short Description**: This particular [branch](https://youtu.be/y4YZHRMmKpE?si=TUfObu4rPeR63bbF&t=1794) of [SPWG](https://docs.google.com/document/d/1K-6V7RXofRxWC3xpRw49beamseJjIQ9CrbA82UPV-8E/edit) (see media announcement [here](https://filecoinfoundation.medium.com/supporting-the-community-the-filecoin-mining-working-group-4bd5f289cbea), twitter [here](https://twitter.com/FilFoundation/status/1405206573606227974?s=20)) has been holding bi-weekly meetings and facilitating overall community communication & cross channel governance engagement since time roughly around Filecoin mainnet launch. In the past 6 months, working group has brought 2 major Filecoin ecosystem participants into governance and group members have been actively participating in FIP discussions and governance as they did in the [past](https://github.com/filecoin-project/FIPs/discussions/554#discussioncomment-5389118), bridging the missing links between core-devs / FF and SPs.
**3. Contributions over past 6 months**: 
- Contribution in FIP discussion that impacted governance. For example, [1](https://github.com/filecoin-project/FIPs/discussions/754#discussioncomment-7970618), [2](https://github.com/filecoin-project/FIPs/discussions/945#discussioncomment-8511900), [3](https://github.com/filecoin-project/FIPs/discussions/943#discussioncomment-8537569), [4](https://github.com/filecoin-project/FIPs/discussions/844#discussioncomment-7214814), [5](https://github.com/filecoin-project/FIPs/discussions/901#discussioncomment-8586779), [6](https://github.com/filecoin-project/FIPs/discussions/854#discussioncomment-7357165), [7](https://github.com/filecoin-project/FIPs/discussions/799#discussioncomment-7594409), [8](https://github.com/filecoin-project/FIPs/discussions/799#discussioncomment-7837547), [9](https://github.com/filecoin-project/FIPs/discussions/799#discussioncomment-7969517), [10](https://github.com/filecoin-project/FIPs/discussions/887#discussioncomment-7859466), [11](https://github.com/filecoin-project/FIPs/discussions/887#discussioncomment-7969156), [12](https://github.com/filecoin-project/FIPs/discussions/887#discussioncomment-7908113), [13](https://github.com/filecoin-project/FIPs/discussions/887#discussioncomment-7953974), [14](https://github.com/filecoin-project/FIPs/discussions/730#discussioncomment-7795861), [15](https://github.com/filecoin-project/FIPs/discussions/512#discussioncomment-7700870), [16](https://github.com/filecoin-project/FIPs/discussions/847#discussioncomment-7248484), [17](https://github.com/filecoin-project/FIPs/discussions/847#discussioncomment-7251285), [18](https://github.com/filecoin-project/FIPs/discussions/847#discussioncomment-7254944), [19](https://github.com/filecoin-project/FIPs/discussions/847#discussioncomment-7736147), [20](https://github.com/filecoin-project/FIPs/discussions/774#discussioncomment-7675863), [21](https://github.com/filecoin-project/FIPs/discussions/833#discussioncomment-7261828), [22](https://github.com/filecoin-project/FIPs/discussions/833#discussioncomment-7108942), [23](https://docs.google.com/document/d/13910NE-O3mUQ6rztt6f3xe7hwW_aS-xaPW_zHuTpBW4/edit#heading=h.4kbcnjlru68f), [24](https://github.com/filecoin-project/FIPs/discussions/901#discussioncomment-8008343), [25](https://github.com/filecoin-project/FIPs/discussions/901#discussioncomment-8008014)
- Participation in FIP proposal and editing. For example, [1](https://github.com/filecoin-project/FIPs/discussions/844), [2](https://github.com/filecoin-project/FIPs/discussions/774), [3](https://github.com/filecoin-project/FIPs/pull/803), [4](https://github.com/filecoin-project/FIPs/pull/788), [5](https://github.com/filecoin-project/FIPs/discussions/712)
- Participating in governance related Slack threads. For instances, [1](https://filecoinproject.slack.com/archives/C0535S9TUUF/p1709084372278169?thread_ts=1709053656.677019&cid=C0535S9TUUF), [2](https://filecoinproject.slack.com/archives/C0535S9TUUF/p1708919288687659), [3](https://filecoinproject.slack.com/archives/C01EU76LPCJ/p1709255066842959?thread_ts=1708560251.653839&cid=C01EU76LPCJ), [4](https://filecoinproject.slack.com/archives/C01EU76LPCJ/p1706236324925419), [5](https://filecoinproject.slack.com/archives/C01EU76LPCJ/p1705471127322829?thread_ts=1705313954.301489&cid=C01EU76LPCJ), [6](https://filecoinproject.slack.com/archives/C01EU76LPCJ/p1702017171167889?thread_ts=1701885098.338279&cid=C01EU76LPCJ), [7](https://filecoinproject.slack.com/archives/C01EU76LPCJ/p1700012261130219?thread_ts=1699988858.879279&cid=C01EU76LPCJ), [8](https://filecoinproject.slack.com/archives/C01EU76LPCJ/p1696645040001679), [9](https://filecoinproject.slack.com/archives/CR98WERRN/p1707120530715319?thread_ts=1707118472.194019&cid=CR98WERRN), [10](https://filecoinproject.slack.com/archives/CR98WERRN/p1702367114487509), [11](https://filecoinproject.slack.com/archives/CR98WERRN/p1700632400170549?thread_ts=1700617266.213279&cid=CR98WERRN)
- Brought two major Filecoin ecosystem participants into governance, creating more engagement.
- Bridge between FF and Asia SP community in forms of bi-weekly meetings and more, lubricating communications and increasing overall governance engagement.  

**4. Github or Twitter**: Unfortunately repo previous under [FilecoinFoundation](https://github.com/FilecoinFoundationWeb) was removed. Now its maintained privately. And the project doesn't have an X account in order to say discrete due to regulatory reasons. But project do have a wx group.
**5. Project Coordinator**: @Fatman13 on both Github and Slack    

![image](https://github.com/filecoin-project/community/assets/1591330/7d1163f3-eb35-437f-b50c-f92cea2ec891)

**Reply by steven004** [reply]
This group is comprised of dedicated Filecoin advocates; it's not easy to consistently hold meetings bi-weekly.

Can anyone else mention a group demonstrating this level of perseverance?  That's why it matters.

**Comment by Kennynauh** [comment]
The project name

Dataswap

A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)

The Dataswap platform offers a groundbreaking, permissionless method for loading DC data onto the Filecoin network.

Through its innovative platform and smart contract development, Dataswap facilitates the loading of public datasets onto the Filecoin network. These contracts enable all participants to access the Filecoin network seamlessly and fairly without requiring permission. Dataswap has also developed data consistency algorithms that accurately verify the consistency between the original dataset and the dataset stored on Filecoin. Additionally, Dataswap contracts effectively manage the distribution and tracking of Datacap, achieving automated management without manual intervention.
you can refer to this link
https://docs.google.com/presentation/d/1Lan7hrBrV1XgDa0V-7g6-TQXwfGl3xVNH90otsDZT9A/edit#slide=id.g1f87997393_0_782

And Dataswap met the requirement about this docment ,https://docs.google.com/document/d/1KLR6nZ8ic4ARj3J46XsxSE_b1RpDP_z3JBKL4alHGGw/edit?pli=1

Through the Dataswap platform, users can conveniently and transparently load and access data on the Filecoin network, driving the process of data exchange and sharing, and actively promoting the development of the Filecoin ecosystem.

The Dataswap project fully open-sourced its code in August 2023.

A link to the project’s GitHub or Twitter
https://github.com/dataswap

Contact info for the project or project lead
https://github.com/waynewyang

**Comment by xianglinhe** [comment]
**1. Project Name:** Power Voting

**2. Project Description:** Power Voting dApp utilizes timelock based on smart contract technology to achieve fair and private voting. Before the voting deadline, no one’s voting results will be seen by others, and the voting process will not be disturbed by other participant’s voting results. After the voting deadline, anyone can count the votes in a decentralized manner, and the results of the counting will executed and stored by smart contract and will not be manipulated by any centralized organization or individual. 

Power Voting dApp has received a grant from FF and will be utilized in the future as a voting governance tool for FF.

**3. Github Links:**
- power voting frontend: https://github.com/black-domain/power-voting
- power voting backend: https://github.com/black-domain/powervoting-backend
- power voting contracts: https://github.com/black-domain/powervoting-contracts
- power oracle contracts: https://github.com/black-domain/power-oracle-contracts
- power oracle node: https://github.com/black-domain/power-oracle-node
- ucan utils: https://github.com/black-domain/ucan-utils


**4. Contact Info:** xianglinhe@gmail.com

**Reply by steven004** [reply]
Interesting tool! 👍
I love this description too: 
_Power Voting dApp` has received a grant from FF` and will be utilized in the future as a voting governance tool for FF._

**Comment by marco-storswift** [comment]
Project:
Lotus

Description of impact in last 6 months:
1 lotus implementers, as part of core dev, is always very active in participating in most FIP discussions and providing suggestions.
2 As a bridge between protocol and user, it can reflect network user needs and ux in protocol design
3 Actively cooperate with the community to implement and test the protocol before an FIP is accepted and launches to the network.
4 In the public core dev call and filecoin slack #fil-implenenter, you can also see that lotus implementers will analyze the scope and risk of each network upgrade to ensure that the NV20/NV21 network upgrade can proceed smoothly.
5 and help coordinate with other implementations, like forest and venus, and implement the test plan

A link to the Github / Twitter:
https://twitter.com/lotus_web3

Contact:
@jennijuju on slack

**Reply by steven004** [reply]
Aha, absolutely.

**Comment by jsoares** [comment]
**The project name**

The _FIP Editors_ group

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

FIP Editors are stewards of the FIP processes, their base responsibilities being defined under [FIP-0001](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md#fip-editor-responsibilities) but their actual involvement far exceeding mere editorial checks and paper pushing. The group's work is not glamorous, but it's essential to the continuous, efficient, and secure evolution of the Filecoin network and protocol. 

While the current set of FIP Editors is employed by PL and FF, none are employed as FIP Editors and pursue this role as voluntary public good work. Jointly, they have provided thousands of review comments and suggestions on [FIP PRs](https://github.com/filecoin-project/FIPs/discussions/960), tend to the pipeline, and contribute to governance discussions, while remaining committed to neutrality and helping improve and advance proposals irrespective of their authorship.

**A link to the project’s GitHub or Twitter**

https://github.com/orgs/filecoin-project/teams/fips-editors

**Contact info for the project or project lead**

No lead; all easy to reach (see link above).

**Reply by jsoares** [reply]
Some notes:
* I am a member of the group, albeit a relevant newcomer. Others have contributed much more over the years.
* I did not consult with the rest of the editors before submitting this nomination, and I cannot imagine anyone joined with the expectation of future rewards. 
* I just find it important to [reward the boring maintenance work on which open source depends](https://stackoverflow.blog/2021/01/07/open-source-has-a-funding-problem/).

**Comment by beng-starboard** [comment]
**Project Name**
Starboard Network Health Dashboard

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

Starboard’s Network Health Dashboard has been contributing essential blockchain analytics to the Filecoin community since its inception. In the last 6 months, Starboard has continued to provide the Filecoin community and the broader public with invaluable analytics and insights into the state of Filecoin. 

Starboard’s analytics and analyses are widely used both internally in the Filecoin community to facilitate collective governance, particularly in FIP discussions to gather and justify support for new initiatives in the Filecoin community (https://github.com/filecoin-project/FIPs/discussions/774). Insights from Starboard continue to be at the centre of Filecoin’s public-facing initiatives such as TLDR (https://filecointldr.io/), Filecoin’s blog (https://filecoin.io/blog/posts/filecoin-news-84/) updates and State of Filecoin reports (https://filecointldr.io/article/state-of-filecoin-2024). 

Starboard’s Network Health Dashboard is known and relied upon outside the Filecoin community as well. Blockchain analytics organisations such as Messari rely on Starboard’s analytics to deliver precise information about Filecoin to the general public (https://messari.io/report/state-of-filecoin-q4-2022).  


**A link to the project’s GitHub or Twitter**
https://github.com/starboard-ventures/
https://twitter.com/starboard_v?lang=en

**Contact info for the project or project lead**
[info@starboardventures.io](mailto:info@starboardventures.io)
[benghau@starboardventures.io](mailto:benghau@starboardventures.io)

**Reply by jennijuju** [reply]
👍 Starboard dashboards are handy to understand network stats! In 2023, their FVM dashboard was SUPER helpful, especially for FVM teams to track FVM's adoption and usage!!

**Comment by kelsien** [comment]
**NOMINATION: (Collective Governance)**

**1. The project name:** 

BlockScience 

  **2. A description of the positive impact of this project for this specific category over the past six months** (Oct 2023 - Mar 2024):
  
In the midst of the FIP0056 governance debates, our timely and accurate analysis played a pivotal role in providing an accurate analysis to evaluate claims being made about the proposed cryptoeconomic sector multiplier mechanism and to inform relevant stakeholders and the broader Filecoin community to navigate towards a resolution and prevent misinformation or misunderstandings.

This analysis and tooling has had a positive impact on Filecoin governance in the past six months via the following:

* Governance Research plus the FIP-0056 debate conclusions was an relevant driver for the[ FIP0001v2 Initiative (#799)](https://github.com/filecoin-project/FIPs/discussions/799) (August 2023 and currently ongoing) its consequent [PR for updating the FIP-0001](https://github.com/filecoin-project/FIPs/pull/850) (October 2023 and currently ongoing). which is an WIP item that has the potential for solving some community demands in terms of further governance scheme improvals, as per the [#945 conversation](https://github.com/filecoin-project/FIPs/discussions/945) initiated by the Asia SP-WG in February 2024.

* Discussions on the general Filecoin Protocol Architecture in terms of its Master Plan (discussion [#725](https://github.com/filecoin-project/FIPs/discussions/725), June to July 2023), as made evident by the [citation](https://github.com/filecoin-project/FIPs/discussions/725#discussioncomment-6299885) of the FIP-0056 debate article for its summarized description of the Filecoin Economy,

* Past work on educating about Termination Fee, as per ["Exploring the Filecoin Termination Fee"](https://medium.com/block-science/exploring-the-filecoin-termination-fee-9351fd40340e) (April 2023) was directly relevant to the governance discussions on the Termination Penalty ([#691](https://github.com/filecoin-project/FIPs/discussions/691), [#712](https://github.com/filecoin-project/FIPs/discussions/712) and [#724](https://github.com/filecoin-project/FIPs/discussions/724) May to July 2023), which then evolved to the discussions related to improving Storage Reliability on FIlecoin ([#843](https://github.com/filecoin-project/FIPs/discussions/843), October 2023),

* Some of the salient points that were made clear on the FIP-0056 are still ongoing, as made evident by the discussion around the Quality Multiplier, such as:
- Discussion on sun-setting FIL+ ([#774, FIP-0080](https://github.com/filecoin-project/FIPs/discussions/774), Aug-Oct 2023)
- "Reasons and Suggestions for Increasing the Quality Multiplier of Filecoin Networks" ([#833](https://github.com/filecoin-project/FIPs/discussions/833), Sep-Nov 2023),

* Work on articulating and educating the on the inner workings of the Baseline Minting was important for the governance discussions on [FIP-0081 (#847](https://github.com/filecoin-project/FIPs/discussions/847), Oct-Dec 2023), as per ["A cadCAD Interactive Calculator to Explore Minting Scenarios in Filecoin"](https://medium.com/block-science/a-cadcad-interactive-calculator-to-explore-minting-scenarios-in-filecoin-284009a2e941) (Mar 2022).

These contributions to Filecoin governance were informed by years of dedicated research, analysis, and development, as well as a strong commitment to innovation, stability, and progress in Filecoin as a critical component of the digital commons. The open nature of outputs creates public goods artifacts that are seminal to informed and engaged collective governance of Filecoin network.

**3. A link to the project’s GitHub or Twitter:**

https://twitter.com/block_science

**4. Contact info for the project or project lead:**

Danilo Lessa Bernardineli, Senior Research Engineer
[Danilo@block.science](mailto:Danilo@block.science)

Nick Hirannet, Chief Operating Officer
[nick@block.science](mailto:nick@block.science)

**Reply by jennijuju** [reply]
💙

**Comment by jennijuju** [comment]
**The project name**

Filecoin Upgrade Coordinators

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

A lot of work happens behind the scenes to ensure a smooth upgrade for each Filecoin upgrade, especially coordinating across FIP governance, core devs (scope and timeline), implementation development, ecosystem integration, documentation, community, network deployment and more. **Filecoin upgrade coordinators act as the cat herders among different network teams and functions and have supported more than ten successful upgrades over the past two years**. They also maintain Network upgrades forums & announcements like https://github.com/filecoin-project/community/discussions/74; status.filecoin.io and so on.

**A link to the project’s GitHub or Twitter**

https://github.com/filecoin-project/community/discussions/74

**Contact info for the project or project lead**

@jennijuju on Filecoin Slack

**Reply by jennijuju** [reply]
Some notes:
- Disclaimer: I have supported Filecoin network upgrades in the past few years. Initially, it was one of the responsibilities under my paid job at PL.  Over the years, as my responsibilities have changed at PL, coordinating network upgrades has become less of a line item under my job description.
-  A lot of "invisible work" is needed to support a network upgrade, and we have developed a high-level process (that is still constantly evolving) to ensure smooth upgrades. 
- Filecoin network upgrades are more than shipping new releases and updating the nodes. There are a lot of risk analysis, ecosystem integration & adoption and critical network infrastructure updates work that differs from upgrade to upgrade, which leads to some network upgrades being more challenging and takes more time to coordinate than others, for example, emergency upgrades to resolve network incidents (2020 chain halt); the FVM upgrades; SDM-or-not-upgrade;  expedited lightening upgrade to resolve chain performance issue & etc
- Filecoin protocol evolves fast, which is great for the network. However, it also makes coordinating (frequent) network upgrades easily a full-time job—and no one in the network is paid to do that specifically. 
💙 Luckily, we have found some volunteers to support this workstream since mid-2022, and they have been a huge help with network upgrade deployment. 
- If you have a high-level understanding of the Filecoin core tech stack and have some PM skills (or want to grow some PM skills), and have been wanting to be A PART OF THE FILECOIN PROTOCOL EVOLVEMENT, I would LOVE to welcome you to join the network upgrade coordinator group and work with us closely! If you are interested in learning more, ping me on Filecoin Slack @jennijuju!

**Reply by Fatman13** [reply]
Isn't upgrade coordination already part of [PL's responsibility](https://docs.google.com/presentation/d/1HnZ3g3N8UCAqEjpwM30edF_kjBpr3TMOyzVxAtkRv5I/edit#slide=id.g2c6522245ff_4_108) as opposed to being public goods?

![image](https://github.com/filecoin-project/community/assets/1591330/1e4cf473-5711-4097-959e-46451637263c)

**Reply by jennijuju** [reply]
This effort is similar to https://vote.optimism.io/retropgf/3/application/0xfcc4798ccc9e2f578ce316278aa40da3fcee3c3f0b92c265062cf31906f2e815, which imho is def a public good effort that hopefully would be contributed by many network teams and individuals (instead of ANY SINGLE GROUP)- the more resources we have here may help escalating and improving filecoin upgrade process.

**Reply by jennijuju** [reply]
Our team (wil be departed from PL soon) is happy to allocate some of our resources to support this network need, and would love to be collaborating with more network teams on this continuous network effort in the long term.

**Reply by momack2** [reply]
As a decentralized network, there is explicitly **no** role or responsibility that must be performed by any specific party. That’s what makes it resilient and self healing - anyone can step up to any role (it’s just OSS collab!). Upgrade coordination across impl teams can be stewarded by anyone. FilOz is grateful to get support from the PL Filecoin Blue Fund to make sure we have some bandwidth to help out here, just like other impl teams get support from groups like the Filecoin Foundation. It takes a whole community to create and improve these public goods! That’s the whole point of retroPGF.

**Reply by beng-starboard** [reply]
The work of the Upgrade Coordinators team is unique and invaluable. Starboard is grateful to the team (especially @jennijuju) for undertaking the tough task of corralling stakeholders and keeping in mind the needs of tooling providers. Recent initiatives such as [FIP0083](https://fips.filecoin.io/FIPS/fip-0083.html) are clear evidence of the team's attentiveness towards our needs. Once again, thank you very much!

**Comment by mjohnson518** [comment]
**Project Name:** Energy Validation Process & Green Scores Methodology

Description:

The [Energy Validation Process (EVP)](https://medium.com/@filecoingreen/embarking-on-the-path-of-sustainability-your-guide-to-the-energy-validation-process-evp-2c11b989a96a) and [Green Scores Methodology](https://green.filecoin.io/Composing_the_Symphony/) represent an effort to incentivize and recognize sustainability within the Filecoin ecosystem. Through a rigorous auditing and validation process, and comprehensive scoring system, these initiatives empower Storage Providers (SPs) to quantify and improve their environmental impact, while also fostering a culture of environmental responsibility across the network.

**Impact:**

- Validation and Recognition: The EVP offers SPs a pathway to demonstrate their commitment to sustainability through data-driven validation. By submitting detailed information on energy consumption, production, and eco-conscious practices, SPs not only receive recognition in the form of an audit output and sustainability claim but also pave the way for possible economic incentives tied to their Green Scores.

- Holistic Impact Assessment: The Green Scores Methodology goes beyond simplistic metrics to provide a nuanced evaluation of SP’s environmental impact. By considering factors such as emissions intensity, location-specific effects on the power grid, and data integrity, this methodology offers a comprehensive assessment that reflects the complex realities of sustainable data storage and Storage Provider operations.

- Incentivizing Sustainability: In the future, we intend to tie Green Scores to economic benefits, such as reputation score boosts and access to green financing opportunities, which will enable the EVP and Green Scores Methodology to incentivize SPs to prioritize sustainability. This will not only benefit individual SPs but also contribute to the broader goal of reducing the environmental impact and advancing Filecoin's commitment to a greener future.

**Why It Matters:**

In an era defined by growing environmental awareness and urgency, initiatives like the EVP and Green Scores Methodology play a crucial role in driving sustainable innovation within the Filecoin ecosystem. By empowering SPs to quantify and improve their environmental impact, while also fostering a culture of transparency and accountability, these projects exemplify the potential of blockchain technology to address pressing global challenges.

**Links:**

- [Green Scores](https://www.greenscores.xyz/)
- [Nova Energy](https://novaenergy.ai/)

**Contacts:**

- [marc@novaenergy.ai](mailto:marc@novaenergy.ai)
- [nima@novaenergy.ai](mailto:nima@novaenergy.ai)

**Comment by dannyob** [comment]
# IPFS Mainnet Conference

## Project name: IPFS Mainnet Conference

## Positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024):

The IPFS Mainnet conference was [a one-day event in Lisbon](https://lu.ma/6suk3oa5), on February 2024, concentrating on the idea of the IPFS public network as a public good. As elements supporting the network — including the bootstrap nodes, the main [[ipfs.io](http://ipfs.io/)](http://ipfs.io) web gateway, and other elements, will now be managed as a commons for the benefit of its users (as opposed to services provided by Protocol Labs, the company), much work needs to be done to ensure their continuing existence.

Creating a place for this discussion, and to plan future steps is an important part of making this transition a success. IPFS Mainnet was notable in that it was initiated and organized, not by the previous custodians of these resources, but by Boris Mann, one of the members of the wider IPFS community.

If IPFS Mainnet is to succeed as a public good, stewarded by the community, events like Mainnet should be encouraged and rewarded: With its use by tools such as Web3 Storage and NFT Storage, and many other Filecoin front ends, IPFS Mainnet has not only been an important resource during the Oct 2023 - Mar 2024, but the planning for its continuing existence in this period leads to greater confidence that it will continue.

## Links

- Social Media: https://lu.ma/6suk3oa5

## Contact info for the project or project lead:

Boris Mann <boris@fission.codes>

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

This week marks an important milestone for the FIL-RetroPGF-1 effort.  Nominations for projects closed on Mar 31, 2024.  We received over 150 nominations for projects across the various categories that were eligible for FIL-RetroPGF-1 funding. 

Thank you to the community for taking the time and effort to put these nominations together. The projects nominated reflect all of the incredible work and progress that the Filecoin community have made together.

As FIL-RetroPGF-1 organizers, our next step is to review the nominations and select eligible projects to apply formally to be considered for FIL-RetroPGF-1 funding. We will be using the criteria outlined [in this document](https://fil-retropgf.notion.site/Nomination-Review-Process-32b62a5ea2cd477f83692cbc233edfda) to determine whether a nominated project should be asked to apply.  Nominated projects will be informed soon, and applications are currently slated to open on Apr 8th.

Thank you, stay tuned, and check back to this page and main our [RetroPGF landing page](https://fil-retropgf.notion.site/FIL-RetroPGF-4b6f5358440043c8bb1bf53f0297541e) for future developments!

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

The FIL-RetroPGF organizers have spent the last week reviewing the nominations submitted in each of the eligible categories. We used the criteria outlined [here](https://fil-retropgf.notion.site/Nomination-Review-Process-32b62a5ea2cd477f83692cbc233edfda) to determine which projects would be invited to apply in this round. Our assessment of eligibility is based on our best interpretation of the information provided in the nomination but tended to bias towards inclusivity, deferring judgement to badgeholders as much as possible. 

The following [page](https://fil-retropgf.notion.site/Round-1-Invited-Applications-3fe95da0c9da4aeba9c6420d1efd0bd6) contains a list of projects that are being invited to apply to be eligible for FIL-RetroPGF-1 funding.

Applications are open from 8 Apr 2024 to 22 Apr 2024. We encourage you to submit your application [here](https://www.fil-retropgf.io/) as soon as possible!

Best regards,
FIL-RetroPGF Organizers

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

This is a gentle reminder that the application submission deadline is fast approaching. If you are a project invited to apply and have not already done so, please submit your applications [here](https://www.fil-retropgf.io/) by Apr 22.

We have updated the [Round 1 - Invited Applications](https://fil-retropgf.notion.site/Round-1-Invited-Applications-3fe95da0c9da4aeba9c6420d1efd0bd6) page to include a POC. Additionally, we have published [the following guidelines](https://fil-retropgf.notion.site/Project-Application-Guidelines-4cf9df76d0f74da9b02cd33f09cbd5b4) on how to approach the application process and included an FAQ at the bottom.

Best Regards,
FIL-RetroPGF Organizers

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

We have extended the deadline to submit your applications. If you haven’t already done so, please submit your application [here](http://fil-retropgf.io/) by 12PM UTC Apr 24, 2024. There will be no further extensions.

Best regards.

**Comment by kkarrancsu** [comment]
Dear Filecoin community,
Thank you for your continued patience. The organizers have finished tallying the results and computing the allocations for each project, according to the scoring rules that were previously set forth. Please visit the following resources for the results:

- [FIL-RetroPGF-1 Results overview](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Results-9ad30232b643460598353489d79b9628)
- [Results Spreadsheet](https://docs.google.com/spreadsheets/d/15tQZU4yfzCCso_dd5NGXfSy6XB2pmrEZzZrG_TEIKyY/edit#gid=1610641018)
- [Deep dive into the results.](https://medium.com/p/7e5a0bcdba08)

We will contact project POCs in the coming days to coordinate distribution. A huge thank you to the entire Filecoin ecosystem for coming together to make this first round successful. Be on the lookout for Round 2!

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

Thank you for your continued patience. We are now in the next phase of FIL-RetroPGF-1—funds distribution. The [Open Impact Foundation](https://openimpact.foundation/) will coordinate distribution and contact projects [that have been allocated funds](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Results-9ad30232b643460598353489d79b9628?pvs=74) from the FIL-RetroPGF-1 process.

Please be on the lookout in your inbox for emails coming from hello@openimpact.foundation

Best Regards,
